doc/classes/PhysicsTestMotionParameters3D.xml

```diff
@@ -14,7 +14,7 @@
 			If set to [code]false[/code], shapes of type [constant PhysicsServer3D.SHAPE_SEPARATION_RAY] are only used for separation when overlapping with other bodies. That's the main use for separation ray shapes.
 		</member>
 		<member name="exclude_bodies" type="RID[]" setter="set_exclude_bodies" getter="get_exclude_bodies" default="[]">
-			Optional array of body [RID] to exclude from collision. Use [method CollisionObject3D.get_rid] to get the [RID] associated with a [CollisionObject3D]-derived node.
+			Optional array [RID]s of bodies in the [PhysicsServer3D] to exclude from collision. Use [method CollisionObject3D.get_rid] to get the [RID] associated with a [CollisionObject3D]-derived node.
```

Comment:
```suggestion
			Optional array of [RID]s of bodies in the [PhysicsServer3D] to exclude from collision. Use [method CollisionObject3D.get_rid] to get the [RID] associated with a [CollisionObject3D]-derived node.
```
These sentences are too clunky though, they should be restored to the original structure and the notes added